### ISSUE TITLE
Add`<main>` landmarks

### ISSUE THREAD

**Issue by Facelessism** [Facelessism/cradle#551]
## Description

Add an appropriate `main` landmark around the primary content of the mini projects that are currently missing one.

**Comment by iamclickify** [comment]
Hi @Facelessism, I can work on this issue. Please assign me. Thanks!
